README.md

```diff
@@ -6,73 +6,147 @@ Welcome to the monorepo of the Liferay Frontend Infrastructure team.
-| -------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
-| [label: `dependencies`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Adependencies) | [New issue](https://github.com/liferay/liferay-frontend-projects/issues/new?labels=dependencies&template=devDependency.md) |
-| [label: `proposal`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Arfc)              | [New issue](https://github.com/liferay/liferay-frontend-projects/issues/new?labels=rfc&template=Proposal.md)               |
-| [label: `question`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Aquestion)         | [New issue](https://github.com/liferay/liferay-frontend-projects/issues/new?labels=question&template=Question.md)          |
+You can [file issues](https://github.com/liferay/liferay-frontend-projects/issues/new/choose) for any of the projects contained in this repository using our predefined templates.
 
-## Projects
+Every project has its own `CONTRIBUTING.md` file explaining how to contribute to the project or how to setup the local development environment but, as a general rule, before starting to work in any new feature by yourself, [file a question issue](https://github.com/liferay/liferay-frontend-projects/issues/new/choose) to discuss it and see if it fits in the backlog.
 
-### `amd-loader`
+For bugs, simply comment in the bug issue that you are working on a fix to avoid double work.
 
-| Package                                        | Issues & PRs                                                                                                                 | Actions                                                                                                               |
-| ---------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------- |
-| [`@liferay/amd-loader`](./projects/amd-loader) | [label: `amd-loader`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Aamd-loader) | [New issue](https://github.com/liferay/liferay-frontend-projects/issues/new?labels=amd-loader&template=amd-loader.md) |
+We follow some guidelines to format [commits](https://github.com/liferay/liferay-frontend-projects/blob/master/guidelines/general/commit_messages.md) and [pull requests](https://github.com/liferay/liferay-frontend-projects/blob/master/guidelines/general/pull_requests.md) so that our `CHANGELOG` files and git history are correctly maintained.
```

Comment:
"git" the executable tool, but "Git" the software project — so I think you want "Git" here.